config.go

```diff
@@ -250,6 +252,19 @@ func ParseConfig(fn string) (Configuration, error) {
 		}
 	}
 
+	removeFromOutput, ok := input.Get("bridge", "remove_from_output")
+	if ok {
+		thingsToRemove := strings.Split(removeFromOutput, ",")
```

Comment:
just to make sure - does this handle spaces? We will have customers who will have:

```
remove_from_output=abcd,  defg
```

remove any extra spaces here too, so we don't field questions as to why `defg` isn't being stripped from the output